test/framework/testdata/prometheus_package_deployment.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: packages.eks.amazonaws.com/v1alpha1
+kind: Package
+metadata:
+  name: generated-prometheus
```

Comment:
might be easier to debug failures in the future if this had a -deployment appended to name to differentiate it from the stateful set when doing k get packages

Reply:
I like the idea, but this is not easy to change without some thorough testing (as many tests referenced this name and they were written to be mode-agnostic).

I created https://github.com/aws/eks-anywhere-packages/issues/712 and https://github.com/aws/eks-anywhere-packages/issues/711 to track this comment. Is it ok I address them in a separate PR?